scripts/site-release.py

```
@@ -861,6 +865,7 @@ def render_install(version, downloads, stars=None):
     body = f"""    <div class="pick">
       <p class="lede">Pick your platform to download the latest build.</p>
 {render_platforms(version, "install")}
+      <p class="recovery"><strong>Updating an older copy?</strong> If the in-app update can’t complete, quit Burrow, download the latest macOS ZIP above, and replace Burrow.app in Applications. Your settings and history stay on this Mac.</p>
```

Ruff (0.16.1)

[warning] String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)